charts/dapr/README.md

```diff
@@ -99,6 +99,7 @@ The Helm chart has the follow configuration options that can be supplied:
 | `global.issuerFilenames.ca`               | Custom name of the file containing the root CA certificate inside the container    | `ca.crt`     |
 | `global.issuerFilenames.cert`             | Custom name of the file containing the leaf certificate inside the container       | `issuer.crt` |
 | `global.issuerFilenames.key`              | Custom name of the file containing the leaf certificate's key inside the container | `issuer.key` |
+| `global.rbac.namespaced`                  | Uses roles and roles bindings in all applicable places other than issuing tokens and listing service accounts for identity verification  | `false` |
```

Comment:
`roles bindings` -> `role bindings`

However not sure I fully understand what this means

Reply:
Probably needs to be rephrased

Reply:
Rephrased